### DISCUSSION TITLE
开启openclash，应该怎么配置外网访问内网服务

### DISCUSSION THREAD

**Discussion by rehack** [vernesong/OpenClash#4100] [Q&A]
我家里网络环境是这样的：
1. PVE主机里安装了Ubuntu服务器（未启动防火墙）
2. Ubuntu里面用docker安装了MySQL等服务
3. iStoreOS作为旁路由，Ubuntu服务器的网关就是旁路由
4. 我用域名做了ddns，主路由器做了MySQL的端口映射

我现在的问题是：
* 当iStoreOS里面的openclash开启，就连不上MySQL服务，不管是Redir-Host模式还是FAKE-IP模式都连不上
* 当关闭iStoreOS里面的openclash，就能连上MySQL服务。

我想实现在公司 通过域名连接家里的MySQL服务 ，并且不要关闭openclash. Ubuntu也要使用openclash
**兄弟们我应该怎么做？怎么配置？**

**Comment by vernesong** [comment]
![image](https://github.com/user-attachments/assets/753e01b6-a776-4b96-80a7-984a243e5a67)端口映射排除掉，经过旁路由clash会nat，或者在旁路由再端口映射一次

**Reply by rehack** [reply]
旁路由的端口映射是这样的吗
![image](https://github.com/user-attachments/assets/9045661f-26fc-46b8-bddb-5279c176bffb)
